packages/phone-number-privacy/signer/src/database/models/request.ts

```diff
@@ -0,0 +1,22 @@
+import { GetBlindedMessageForSaltRequest } from '../../salt-generation/get-salt'
+
+export const REQUESTS_TABLE = 'requests'
+export enum REQUESTS_COLUMNS {
+  address = 'address',
```

Comment:
nit: maybe we can call this "caller_address" to make it clear who's address this is